testing/src/test/java/io/stargate/it/BaseOsgiIntegrationTest.java

```diff
@@ -283,129 +242,28 @@ Object proxy(
     return (T) proxy(p, p.getClass().getClassLoader(), clazz.getClassLoader(), clazz);
   }
 
-  /** Starts a docker backend for the persistance layer */
-  GenericContainer startBackend() throws IOException, InterruptedException {
-    boolean skipHostNetworking = Boolean.getBoolean(SKIP_HOST_NETWORKING_FLAG);
-
-    GenericContainer backend;
-    if (skipHostNetworking) {
-      backend =
-          new FixedHostPortGenericContainer<>(dockerImage)
-              .withFixedExposedPort(7000, 7000)
-              .withEnv("DS_LICENSE", "accept")
-              .waitingFor(
-                  Wait.forLogMessage(".*Starting listening for CQL clients.*", 1)
-                      .withStartupTimeout(java.time.Duration.of(120, ChronoUnit.SECONDS)))
-              .withEnv("BROADCAST_ADDRESS", "127.0.0.1")
-              .withEnv("CASSANDRA_BROADCAST_ADDRESS", "127.0.0.1");
-    } else {
-      backend =
-          new GenericContainer<>(dockerImage)
-              .withEnv("DS_LICENSE", "accept")
-              .waitingFor(
-                  Wait.forLogMessage(".*Starting listening for CQL clients.*", 1)
-                      .withStartupTimeout(java.time.Duration.of(120, ChronoUnit.SECONDS)))
-              .withNetworkMode("host")
-              .withEnv("LISTEN_ADDRESS", "127.0.0.2")
-              .withEnv("CASSANDRA_LISTEN_ADDRESS", "127.0.0.2");
-    }
-
-    backend.start();
-    backend.withLogConsumer(
-        new Slf4jLogConsumer(LoggerFactory.getLogger("docker")).withPrefix("docker"));
-
-    logger.info("{} backend started", dockerImage);
-
-    // Setup for graphql test here so that when the graphql service starts up it'll know about the
-    // schema changes
-    backend.execInContainer(
-        "cqlsh",
-        "-e",
-        "CREATE KEYSPACE IF NOT EXISTS betterbotz with replication={'class': 'SimpleStrategy', 'replication_factor':1};");
-    backend.execInContainer(
-        "cqlsh",
-        "-e",
-        "CREATE TABLE IF NOT EXISTS betterbotz.products (id uuid,name text,description text,price decimal,created timestamp,PRIMARY KEY (id, name, price, created)); ");
-    backend.execInContainer(
-        "cqlsh",
-        "-e",
-        "CREATE TABLE IF NOT EXISTS betterbotz.orders (id uuid, prod_id uuid, prod_name text, description text, price decimal, sell_price decimal, customer_name text, address text, PRIMARY KEY (prod_name, customer_name));\n");
-
-    return backend;
-  }
-
   @Before
-  public void baseSetup() throws BundleException, InterruptedException, IOException {
-    if (backendContainer != null && !backendContainer.getDockerImageName().equals(dockerImage)) {
-      logger.info("Docker image changed {} {}", dockerImage, backendContainer.getDockerImageName());
-
-      if (stargateStarter != null) stargateStarter.stop();
-
-      backendContainer.stop();
-
-      backendContainer = null;
-      stargateStarter = null;
+  @BeforeEach
```

Comment:
`@Before` and `@BeforeEach`?

Reply:
That is required because some of the sub-classes use JUnit 4, others JUnit 5... we'll sort it out in a follow-up PR.

Reply:
Got it, thanks